src/transformers/pipelines/depth_estimation.py

```diff
@@ -112,11 +112,11 @@ def postprocess(self, model_outputs):
         )
 
         formatted_outputs = []
-        for predicted_depth in outputs:
-            depth = predicted_depth.detach().cpu().numpy()
+        for output in outputs:
+            depth = output["predicted_depth"].detach().cpu().numpy()
             depth = (depth - depth.min()) / (depth.max() - depth.min())
             depth = Image.fromarray((depth * 255).astype("uint8"))
 
-            formatted_outputs.append({"predicted_depth": predicted_depth, "depth": depth})
+            formatted_outputs.append(output | {"depth": depth})
```

Comment:
At the moment we support Python 3.8, however `|` for `dict` joining was added in Python 3.9, so we have to avoid it.